student.rb

```diff
@@ -0,0 +1,11 @@
+require('./person')
+class Student < Person
+  def initialize(classroom, age, parent_permission, name)
```

Comment:
- [ ] Kindly set the default `name` to `'Unknown'` and default `parent_permission` to `true`